plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/RollbackToSnapshotProcedure.java

```diff
@@ -31,6 +31,7 @@
 import static java.lang.invoke.MethodHandles.lookup;
 import static java.util.Objects.requireNonNull;
 
+@Deprecated
```

Comment:
Is this part of a larger endeavor ?
If there are other follow-ups maybe it is worth creating a epic issue to track what else will be done.